cashu/wallet/api/router.py

```diff
@@ -18,8 +18,10 @@
 from ...wallet.nostr import receive_nostr, send_nostr
 from ...wallet.wallet import Wallet as Wallet
 from .api_helpers import verify_mints
+from .error_handler import ErrorHandler
 
 router: APIRouter = APIRouter()
+router.route_class = ErrorHandler
```

Comment:
Hrm, did you consider a [Middleware](https://fastapi.tiangolo.com/tutorial/middleware/)? I suppose switching out the Route class would work too but a middleware is 'more normal'